### ISSUE TITLE
утилита вылетает с ошибкой

### ISSUE THREAD

**Issue by generalissimus1966** [Etersoft/p12export#1]
Я попытался использовать вашу утилиту, и она вылетает с ошибкой, никаких выходных файлов не создаёт
Версия КриптоПро 5.0.11553 КС1
Версия ОС Windows 2012 Server R2

C:\Users\build\Desktop\ex>p12export.exe
Selected certificate:
Subject: "build SignB"
Issuer: "GOST2001 Lynx Test CA"
Not valid before: 17.02.2020 11:05:34
Not valid after: 17.05.2021 12:05:35
line 980: CryptExportKey, error 0x8009000b (2148073483)

С разными сертификатами номер строки разный, номер ошибки одинаковый. Не зависит от того, запускал под администратором, или нет.

**Comment by vitlav** [comment]
Подозреваю, что это
У вас нет разрешений на экспорт ключа, потому что при создании ключа не был установлен соответствующий флаг. Ошибка 0x8009000B (-2146893813)
https://www.kontur-extern.ru/info/oshibka-kopirovaniya-konteynera

**Comment by generalissimus1966** [comment]
Да, код ошибки такой, но он возникает со всеми сертификатами, независимо от
того, был установлен флаг "экспортируемый", или нет. я сам генерировал эти
сертификаты, если что.

сб, 3 окт. 2020 г. в 16:57, Vitaly Lipatov <notifications@github.com>:
